platform/app/src/routes/index.tsx

```diff
@@ -15,15 +15,24 @@ import PrivateRoute from './PrivateRoute';
 const bakedInRoutes = [
   {
     path: '/debug',
-    children: Debug,
+    children: DataSourceWrapper,
+    props: {
```

Comment:
can we have some doc about who is child of whom since there are two children props here, thanks

Reply:
What do you mean by two children? I'm just configuring the wrapper to be the parent of any route since any route could retrieve or access data from a data source, that's all

Reply:
I know what you are doing, but for someone who reads the code it might not be evident. So some docs would be good since there are two children props

Reply:
I'm still not sure what you mean by two children, I can only see one for the route and another for the children. I'm also not sure yet why we should explain such technical detail about having a wrapper for the routes. If the data source wrapper is already documented I think that's all people need to know? Again, I'm open to suggestions on where I could add this information